group_vars/nomad_cluster.yml

```diff
@@ -1,12 +1,13 @@
+---
 consul_version: '1.20.3'
 consul_gossip_encryption_key: '{{ vault_consul_gossip_encryption_key }}'
 consul_acl_management_token: "{{ vault_consul_acl_management_token }}"
-nomad_server_group: 'nomad_servers_production'
+nomad_server_group: nomad_servers_{{ runtime_env | default('sandbox') }}
```

Comment:
I think this should still be `nomad_servers_production`

Reply:
I was reminded by @acozine that this has nothing to do with inventory.